src/Ring-Core/RGPackage.class.st

```diff
@@ -43,6 +43,16 @@ RGPackage >> addoptToParentStub [
 	^ self parent addPackage: self.
 ]
 
+{ #category : #accessing }
+RGPackage >> buildMicroDownUsing: aMicMicrodownTextualBuilder withComment: aString [ 
```

Comment:
Same as before. Should this be an extension method?

Reply:
Done